src/app/modules/main/chat_section/chat_content/controller.nim

```diff
@@ -107,6 +107,27 @@ method init*(self: Controller) =
     # remove from pinned messages model
     self.delegate.onUnpinMessage(args.messageId)
 
+  self.events.on(SIGNAL_COMMUNITY_CHANNEL_EDITED) do(e:Args):
+    let args = CommunityChatArgs(e)
+    if(self.chatId != args.chat.id or not self.belongsToCommunity):
+      return
+
+    if (args.communityId == self.sectionId):
+      let chatDto = ChatDto(
```

Comment:
I think it might simplify if the service converted the CommunityChat to the ChatDto itself, since I see you have two controllers that have to do the same job.